script/Arcueid/Day 13/13_00_ARC13_6.txt

```diff
@@ -131,17 +131,17 @@ His lower body grows back rapidly.
 [sha:14d21da6b8e7b56afb50f9cb13660c1028ea1277]{
 -- Page 455, Offset 32988.
 -- あれほど瓦礫を濡らしていた血液が、巻き戻し映像のようにソレの体に吸い込まれていく。
-The blood that soaked the rubble gets absorbed back into his body like a video being rewinded.
+The blood that soaked the rubble is absorbed back into his body, like a video being rewinded.
```

Comment:
Like a tape on rewind, the blood that soaked the rubble is absorbed back into his body.